image/verify/src/verifier.rs

```diff
@@ -112,10 +112,32 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
         let image_info = okref(&image_info)?;
 
         // Verify FMC
-        let (fmc_info, fmc_log_info) = self.verify_fmc(image_info.fmc, reason)?;
+        let fmc_info = self.verify_fmc(image_info.fmc, reason)?;
 
         // Verify Runtime
-        let (runtime_info, rt_log_info) = self.verify_runtime(image_info.runtime)?;
+        let runtime_info = self.verify_runtime(image_info.runtime)?;
+
+        // The FMC and RT FW images within the bundle each include an SVN, for
```

Comment:
Suggest putting this logic in a function (eg verify_svn) that can return the effective fuse svn on success.

Reply:
Have factored this into a helper function. I think it's best to leave the effective fuse SVN calculation as its own helper, since those are different tasks.